### PR TITLE
Tests for paging up to find old messages

### PR DESCRIPTION
**Pull request by andybalaam**
Part of https://github.com/vector-im/element-web/issues/25449

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: cypress/e2e/read-receipts/high-level.spec.ts [hunk 1]
```diff
@@ -2344,12 +2369,236 @@ describe("Read receipts", () => {
...
+    }
+
+    function assertMessageNotLoaded(messagePart: string) {
+        cy.get(".mx_EventTile_body").contains(messagePart).should("not.exist");
+    }
+
+    function pageUp() {
+        cy.get(".mx_RoomView_messagePanel").then((refs) =>
+            refs.each((_, messagePanel) => {
+                console.log(`AJB ${messagePanel.scrollTop}`);
```
Comment (weeman1337) [inline cypress/e2e/read-receipts/high-level.spec.ts:2383]: Should these still be here? Eventually convert hem to common log messages.

File: cypress/e2e/read-receipts/high-level.spec.ts [hunk 2]
```diff
@@ -2344,12 +2369,236 @@ describe("Read receipts", () => {
...
+            pageUp();
+
+            // Then the room remains read
+            assertStillRead(room2);
+            assertReadThread("Root1");
+            assertReadThread("Root2");
+            assertReadThread("Root3");
+        });
+        // XXX: fails because we see a dot instead of an unread number - probably the server and client disagree
+        it.skip("After marking room as read based on a thread message, opening threads view to find old threads that were never read leaves the room read", () => {
```
Comment (weeman1337) [inline cypress/e2e/read-receipts/high-level.spec.ts:2570]: I assume all the skipped tests are what we need to fix?

**Review by weeman1337 — APPROVED** [review]
More tests :rocket: Looks good!

File: cypress/e2e/read-receipts/high-level.spec.ts (hunk 1 above)
Comment (andybalaam) [reply cypress/e2e/read-receipts/high-level.spec.ts]: Thank you, accidentally left in. Now deleted.

File: cypress/e2e/read-receipts/high-level.spec.ts (hunk 2 above)
Comment (andybalaam) [reply cypress/e2e/read-receipts/high-level.spec.ts:2568]: Yes, everything with "XXX: fails" is a bug we need to fix.

**Pull request merged**